NineChronicles.Headless/GraphTypes/StandaloneQuery.cs

```diff
@@ -54,7 +54,8 @@ public StandaloneQuery(StandaloneContext standaloneContext, IConfiguration confi
 
                     return new StateContext(
                         chain.ToAccountStateGetter(blockHash),
-                        chain.ToAccountBalanceGetter(blockHash)
+                        chain.ToAccountBalanceGetter(blockHash),
+                        chain.Tip.Index
```

Comment:
Even when a block hash is given as an argument, but block index always use latest index. it may cause confusion.

Reply:
I submitted a new commit to address the issue to use the right block index.